### DISCUSSION TITLE
If owner is null, owner.get() throws null pointer exception

### DISCUSSION THREAD

**Discussion by arvind-das** [orientechnologies/orientdb#9865] [General] [answered]
Even if the method is marked as deprecated, it is referenced in some other ORM tools like ferma, this method gets called and throws null pointer exception, can we fix this ? I can raise a PR

https://github.com/orientechnologies/orientdb/blob/901bc89c48caf8c6d8829e910897bd0e9ef25877/core/src/main/java/com/orientechnologies/orient/core/record/impl/ODocument.java#L1910

**Comment by tglman** [comment, marked as answer]
Hi,

Yep seems so, feel free to do a PR!
